### ISSUE TITLE
asplashscreen

### ISSUE THREAD

**Issue by joolswills** [RetroPie/RetroPie-Setup#1386]
Look into splashscreen videos again (convert to systemd service?) - omxplayer sometimes starts before system is ready (error is - waiting for dbus address to appear). Need another solution for the hacky waiting for /tmp to be writable.

**Comment by joolswills** [comment]
Although some other users reported this here

http://blog.petrockblock.com/forums/topic/splash-video-not-loading-anymore/

I am currently unable to reproduce. Video is showing fine each time.

**Comment by joolswills** [comment]
Scrap that - I did just get it to fail.

**Comment by joolswills** [comment]
So I think what was happening:

We were incorrectly waiting for /tmp to be writable (which seemed to work before), and then starting asplashscreen. We really just wanted to make sure dbus is running as else omxplayer complains. On some boots with the wait for /tmp dbus wasn't actually running - which caused omxplayer to start outputting errors. I believe this triggers systemd to disable the init.d script from startup since it doesn't finish in a required amount of time.

We now wait instead for the "dbus" process. I have rebooted lots of times now, and it still seems to be showing. I also had to re-enable the splashscreen service for it to start working again (via retropie-setup)
